frontend/src/components/Login.jsx

```diff
@@ -0,0 +1,60 @@
+import { useRef } from "react";
+
+const Login = ({ setCurrUser, setShow }) => {
```

Comment:
@omkogde-vf Avoid sending setters through components and grand components, use ContextApi/useContext for the same.